bird_classify.py

```diff
@@ -101,43 +103,74 @@ def main():
     'alarm' if a defined label is detected."""
     args = user_selections()
     print("Loading %s with %s labels."%(args.model, args.labels))
-    engine = ClassificationEngine(args.model)
-    labels = load_labels(args.labels)
+    interpreter = make_interpreter(args.model)
+    interpreter.allocate_tensors()
+    labels = read_label_file(args.labels)
+    output_tensors_size = interpreter.get_output_details()[0]['shape'][0]
+
+    if output_tensors_size != 1:
+        raise ValueError(
+            ('Classification model should have 1 output tensor only!'
+             'This model has {}.'.format(output_tensors_size)))
```

Comment:
Looks like `output_tensors_size` is an element of first tensor shape, not the the number of output tensors. The number of output tensors is probably `len(interpreter.get_output_details())`.